packages/styles/src/skeleton.scss

```diff
@@ -0,0 +1,48 @@
+
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-skeleton;
+
+.#{$block} {
+  @include fd-reset();
+
+  display: inline-block;
+  cursor: wait;
+
+  &__canvas {
+    display: block;
+    fill: var(--sapContent_Placeholderloading_Background);
+  }
+
+  &__middle {
+    stop-color: var(--fdSkeletonGradientMiddleColor);
+    animation: fd-skeleton-shimmer 2s linear infinite;
+  }
+
+  &--animated {
```

Comment:
I don't get this modifier class. Shouldn't the `fd-skeleton` be always animated? I couldn't find an example with just `fd-skeleton` class. It's always `<div class="fd-skeleton fd-skeleton--animated" style="...">`.
In this case do we need this extra modifier?

Reply:
merged it with the block